internal/cmd/skupper/connector/kube/connector_create.go

```diff
@@ -271,25 +283,54 @@ func (cmd *CmdConnectorCreate) WaitUntil() error {
 		return nil
 	}
 
+	if cmd.status == "none" {
+		return nil
+	}
+
 	waitTime := int(cmd.timeout.Seconds())
+
+	var connectorCondition *metav1.Condition
+
 	err := utils.NewSpinnerWithTimeout("Waiting for create to complete...", waitTime, func() error {
 
 		resource, err := cmd.client.Connectors(cmd.namespace).Get(context.TODO(), cmd.name, metav1.GetOptions{})
 		if err != nil {
 			return err
 		}
 
-		if resource != nil && resource.IsConfigured() {
+		isConditionFound := false
+		isConditionTrue := false
+
+		switch cmd.status {
+		case "ready":
```

Comment:
As I recall, for sites the default was to wait for ready unless configured was specified explicitly. Is there a reason this is switched here?

Reply:
I thought that for connectors/listeners the default should be "configured" given that they are not ready unless there is match; otherwise it would timeout always. Am I wrong?

Reply:
No, that does indeed make sense!